src/InvokePlaster.ps1

```diff
@@ -24,10 +24,13 @@ Please follow the scripting style of this file when adding new script.
     General notes
 #>
 function Invoke-Plaster {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage('PSAvoidUsingWriteHost', '', Scope='Function')]
+    [System.Diagnostics.CodeAnalysis.SuppressMessage('PSAvoidShouldContinueWithoutForce', '', Scope='Function', Target='CopyFileWithConflictDetection')]
```

Comment:
Sorry, I wasn't very clear.  This is the PR on PSSA I was referring to: https://github.com/PowerShell/PSScriptAnalyzer/pull/625  I was running into this issue in Invoke-Plaster because I had several helper functions within `begin` that use $PSCmdlet.ShouldProcess.